src/main/java/de/doubleslash/keeptime/model/Model.java

```diff
@@ -76,6 +77,9 @@ public Model(final ProjectRepository projectRepository, final WorkRepository wor
    public final ObjectProperty<Boolean> useHotkey = new SimpleObjectProperty<>(false);
    public final ObjectProperty<Boolean> displayProjectsRight = new SimpleObjectProperty<>(false);
    public final ObjectProperty<Boolean> hideProjectsOnMouseExit = new SimpleObjectProperty<>(true);
+   public final ObjectProperty<Boolean> emptyNoteReminder = new SimpleObjectProperty<>(false);

```

Comment:
Name of the variable is not nice. As it is a boolean value it should 'answer a question' lik e.g. hintIfNoteIsEmpty -> yes or no :)